backports/gflags.v3.8/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Florent Ferreri <florent@seqsense.com>
+pkgname=gflags
+pkgver=2.2.2
+pkgrel=0
+pkgdesc="The gflags package contains a C++ library that implements commandline flags processing."
+url="https://github.com/gflags/gflags"
+arch="noarch"
+license="BSD-3-Clause"
+options="!check"
+makedepends="cmake"
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/gflags/$pkgname/archive/refs/tags/v${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+    mkdir build
+    cd build
+    cmake -DCMAKE_INSTALL_PREFIX=/usr ..
```

Comment:
If there is no requirement for `CMAKE_BUILD_TYPE`, `-DCMAKE_BUILD_TYPE=none` is recommended.
https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/4317